app/helpers/view_components/mentor/header.rb

```diff
@@ -35,10 +35,11 @@ def bottom
 
       def stats
         tag.div(class: "stats") do
-          # TODO: (Required)
-          tag.div("131 solutions mentored", class: "stat") +
-            # TODO: (Required)
-            tag.div("95% satisfaction", class: "stat")
+          tag.div("#{current_user.num_solutions_mentored} solutions mentored", class: "stat") +
+            (if current_user.mentor_satisfaction_percentage
+               tag.div("#{current_user.mentor_satisfaction_percentage}% satisfaction",
+                 class: "stat")
+             end)
```

Comment:
Is this an acceptable way of doing this?

Reply:
If it works!

I'd probably have switched to `safe_join` for readiability:
```ruby
safe_join(
  [
    tag.div("#{current_user.num_solutions_mentored} solutions mentored", class: "stat"),
    (tag.div("#{current_user.mentor_satisfaction_percentage}% satisfaction", class: "stat") if current_user.mentor_satisfaction_percentage)
  ].compact
)
```

But it all sort of depends on line-length as to what works.

Reply:
rubocop rewrites that to:

```ruby
safe_join(
            [
              tag.div("#{current_user.num_solutions_mentored} solutions mentored", class: "stat"),
              (if current_user.mentor_satisfaction_percentage
                 tag.div("#{current_user.mentor_satisfaction_percentage}% satisfaction",
                   class: "stat")
               end)
            ].compact
          )
```

Which I still like better than my version

Reply:
OK :)